packages/EasyErrorHandler/src/Bridge/Symfony/DependencyInjection/Compiler/ApiPlatformCompilerPass.php

```diff
@@ -5,13 +5,24 @@
 namespace EonX\EasyErrorHandler\Bridge\Symfony\DependencyInjection\Compiler;
 
 use EonX\EasyErrorHandler\Bridge\BridgeConstantsInterface;
+use EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\TraceableErrorHandlerInterface;
+use EonX\EasyErrorHandler\Bridge\Symfony\TraceableErrorHandler;
+use EonX\EasyErrorHandler\Interfaces\ErrorHandlerInterface;
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
 
 class ApiPlatformCompilerPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container): void
     {
+        if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
```

Comment:
This has nothing to do with ApiPlatform, it shouldn't be in this compiler pass.

Reply:
`EasyErrorHandlerSymfonyCompilerPass` naming will be ok here?

Reply:
Done it with `ErrorHandlerCompilerPass` name